resources/packs/vcd/provider.go

```diff
@@ -0,0 +1,99 @@
+package vcd
+
+import (
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+	"go.mondoo.com/cnquery/resources"
+)
+
+func (v *mqlVcd) GetProviderVDCs() ([]interface{}, error) {
+	op, err := vcdProvider(v.MotorRuntime.Motor.Provider)
+	if err != nil {
+		return nil, err
+	}
+	client := op.Client()
+
+	vdcList, err := client.QueryProviderVdcs()
+	if err != nil {
+		return nil, err
+	}
+
+	list := []interface{}{}
+	for i := range vdcList {
+		entry, err := newMqlVcdProvider(v.MotorRuntime, vdcList[i])
+		if err != nil {
+			return nil, err
+		}
+		list = append(list, entry)
+	}
+
+	return list, nil
+}
+
+func newMqlVcdProvider(runtime *resources.Runtime, vcdProvider *types.QueryResultVMWProviderVdcRecordType) (interface{}, error) {
+	return runtime.CreateResource("vcd.vdcProvider",
```

Comment:
Not sure, whether this should be "vdc" or "vcd". The text mentions "vdcProvider", but the var is named "vcdProdiver".

Reply:
`vcd.vdcProvider` is actually correct.